src/rgw/radosgw-admin/radosgw-admin.hpp

```diff
@@ -0,0 +1,89 @@
+#ifndef RADOSGW_ADMIN_HPP 
+ #define RADOSGW_ADMIN_HPP 1
```

Comment:
I don't necessarily think every source file needs its own header (some of them may not really need anything unique, but will need to share with other headers)-- however, I can also go with that idea (we'll just at least potentially have some headers that do nothing, or just include one other header).

This particular new header file here is indeed a giant bag-- I expect that as I start doing more restructuring we'll naturally end up with more and different organization (and new source files), but since there is a LOT in here-- both in terms of line count and conceptualization-- it will have to happen incrementally.